src/component/typingbox/__tests__/TypingBox.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import TypingBox from '../TypingBox';
+import renderer from 'react-test-renderer';
+import { mount, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+
+describe("TypingBox", () => {
+  it("renders a textbox", () => {
+    const TypingBoxComponent = renderer.create(<TypingBox/>).toJSON();
+    expect(TypingBoxComponent).toMatchSnapshot();
+  });
+
+  it("renders correctly with no input", () => {
```

Comment:
I might change this wording slightly to reflect what we are testing.

`The input field is empty when we pass an empty value`